website/src/app/kb/user-guides/android-client/readme.mdx

```diff
@@ -14,8 +14,43 @@ Google Play Store.
 
 ## Installation
 
-[Download the Firezone app](https://play.google.com/store/apps/details?id=dev.firezone.android)
-from the Google Play Store.
+1. [Download the Firezone app](https://play.google.com/store/apps/details?id=dev.firezone.android)
+   from the Google Play Store.
+1. Run the app. The welcome screen will show "Enable VPN Permission" and the
+   Firezone logo.
+1. Tap `Request Permission`. Android will show a dialog saying,
+   `Connection request`.
+1. Tap `OK`. Firezone will switch to the `Signed Out` screen.
+
+## Usage
+
+### Signing in
+
+1. Tap `Sign In`. Firezone will open a sign-in page in your web browser.
+1. Select your account and sign in. Firezone will show your username and
+   Resources.
+
+### Accessing a Resource
+
+When Firezone is signed in, web browsers and other programs will automatically
+use it to securely connect to Resources.
+
+To copy-paste the address of a Resource:
+
+1. Tap on the Resource.
+1. Tap on the address, e.g. `example.com`. Firezone will copy the address to the
+   clipboard.
+1. Open your web browser, such as Google Chrome.
+1. Tap `Link your copied` at the top, or the paste button above the keyboard.
+1. Navigate to the link.
```

Comment:
I don't know what to call this, it's just an arrow pointing right in the bottom-right of the keyboard. I can say that for English and I guess if we do other languages and it moves around we can deal with it then? It's not labelled "Go" or "Enter"

Reply:
Yeah and it will be different for different keyboards. Added a suggestion that might work universally.